.changes/unreleased/tinymce-TINY-11658-2025-02-06.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: '`tox-toolbar-textfield` didn''t have a focus style.'
```

Why do we have those single quotes, I believe it should be like this:

```suggestion
body: `tox-toolbar-textfield` didn't have a focus style.
```

---

changed